pkg/chaosctl/cmd/recover.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/chaos-mesh/chaos-mesh/pkg/chaosctl/common"
+	"github.com/chaos-mesh/chaos-mesh/pkg/chaosctl/recover"
+	ctrlclient "github.com/chaos-mesh/chaos-mesh/pkg/ctrl/client"
+	"github.com/chaos-mesh/chaos-mesh/pkg/label"
+)
+
+type RecoverOptions struct {
+	namespace string
+	labels    *[]string
+}
+
+func NewRecoverCommand(logger logr.Logger, builders map[string]recover.RecoverBuilder) (*cobra.Command, error) {
+	o := &RecoverOptions{namespace: "default"}
+
+	recoverCmd := &cobra.Command{
+		Use:   `recover (CHAOSTYPE) (PODs) [-n NAMESPACE]`,
```

Comment:
It is rare to use (...) in this situation, for `PODS` you can provide `pod[,pod[,pod...]]`.

Reply:
Reasonable, modified.